pytest_rts/tests/evaluation/evaluation_utils.py

```diff
@@ -98,3 +102,65 @@ def print_remove_test_output(
     )
     logger.info(f"STORING TO DATABASE: {db_name}")
     logger.info("============")
+
+
+def get_test_suite_size():
+    """Query how many tests are in mapping database"""
+    conn = sqlite3.connect(DB_FILE_NAME)
+    size = int(conn.execute("SELECT count() FROM test_function").fetchone()[0])
+    conn.close()
+    return size
+
+
+def get_all_tests_for_srcfile(file_id):
+    """Query all tests for a source code file"""
+    conn = sqlite3.connect(DB_FILE_NAME)
+    tests = []
+    data = conn.execute(
+        """ SELECT DISTINCT context
+                FROM test_function
+                JOIN test_map ON test_function.id == test_map.test_function_id
+                WHERE test_map.file_id = ? """,
+        (file_id,),
+    )
+    for line in data:
+        test = line[0]
+        tests.append(test)
```

Comment:
you don't need a loop here; use comprehension. Also do it after you close the connection, right in `return`.